commands/azureCommands/delete-image.ts

```diff
@@ -36,7 +36,7 @@ export async function untagAzureImage(context?: AzureImageTagNode): Promise<void
     const shouldDelete = await ext.ui.showWarningMessage(
         `Are you sure you want to untag '${image.toString()}'? This does not delete the manifest referenced by the tag.`,
         { modal: true },
-        DialogResponses.deleteResponse,
+        { title: "Untag" },
         DialogResponses.cancel);
 
     if (shouldDelete === DialogResponses.deleteResponse) {
```

Comment:
That ain't gonna work - you're comparing the result to deleteResponse.  Preferred pattern:

const untag = { title: "Untag" };
const shouldUntag = await ext.ui.showWarningMessage(
…
        untag, 
...
);

      if (shouldDelete === untag) {

Reply:
Sorry about that one. 
Fixed